### PR TITLE
Update readme, including npm install

### PR DESCRIPTION
**Pull request by dusan-bc**
Forgot to include NPM INSTALL, so changed the readme to reflect that, please review.

### CODE REVIEW

File: supportive_cases/zos-connect/README.md [hunk 1]
```diff
@@ -7,16 +7,25 @@ This folder contains a test sample that triggers an API endpoint through z/OS Co
...
+        npm run setTest4z
+      
+3. Run the following command and enter the z/OS Connect specific information. 
+   Ask your mainframe administrator for the information.
+   **In case you do not have the username and passwor, you can leave them blank**
+
+        npm run setZosConnect
 
-2. Navigate to src/test/zosConnectSample and change the values of mainDataset and copybook to reflect your environment. We are using the CICS Catalog Manager example application. 
+4. Navigate to [src/test/zosConnectSample] and change the values of mainDataset and copybook to reflect your environment. We are using the CICS Catalog Manager example application.
```
Comment (vibhor-bc) [inline supportive_cases/zos-connect/README.md:28]: Do we have a link on IBM website somewhere for this example application which can be added here ?

Comment (dusan-bc) [reply supportive_cases/zos-connect/README.md]: included

Comment (vibhor-bc) [reply supportive_cases/zos-connect/README.md]: Looks good. Thanks

**Review by vibhor-bc — APPROVED** [review]

File: supportive_cases/zos-connect/README.md [hunk 2]
```diff
@@ -7,16 +7,25 @@ This folder contains a test sample that triggers an API endpoint through z/OS Co
...
+
+2. Run the following command and enter the Test4z specific information. 
+   Ask your mainframe administrator for the information.
+   **You can skip this step if you ran this command before**
 
+        npm run setTest4z
+      
+3. Run the following command and enter the z/OS Connect specific information. 
+   Ask your mainframe administrator for the information.
+   **In case you do not have the username and passwor, you can leave them blank**
```
[the remark this answers is not in the document]
Comment (dusan-bc) [reply supportive_cases/zos-connect/README.md:24]: will fix

**Pull request merged**